HordaMaria/HW7/task3

```diff
@@ -0,0 +1,8 @@
+def count_char(input_str):
```

Comment:
good job
look at this code
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result